frontend/src/locales/en.yml

```diff
@@ -989,6 +1003,26 @@ archive:
   project-search-bar-placeholder: Search @.lower:{'common.project'} name
   instance-search-bar-placeholder: Search @.lower:{'common.instance'} name
   environment-search-bar-placeholder: Search @.lower:{'common.environment'} name
+deployment-config:
+  stage-n: Stage {n}
+  add-selector: Add selector
+  update-success: Successfully updated deployment config.
+  this-is-example-deployment-config: This is the deployment config example. You need to edit and save it.
+  n-databases: '{n} database | {n} databases'
+  selectors: Selectors
+  wont-be-deployed: Will no be deployed
+  add-stage: Add stage
+  name-placeholder: Stage name...
+  error:
+    at-least-one-stage: At least 1 stage
+    stage-name-required: Stage name is required
+    at-least-one-selector: At least 1 selector in each stage
+    env-in-selector-required: '"bb.environment In" is required in each stage'
```

Comment:
we should not expose bb.environment. This is not user comprehensive. Instead we should translate to environment from the UI

Reply:
I prefer to make it "environment" here, and hide "bb." prefix in every Label-related components ( label selector for example). 
After that I may introduce another UI refactor to convert labels as normal properties.

Reply:
updated, PTAL